apps/cns-website/src/app/pages/people-profile/components/contact-info/contact-info.component.scss

```diff
@@ -0,0 +1,65 @@
+@use 'vars' as vars;
+
+:host {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+
+  @media (max-width: 639px) {
+    min-height: auto;
+  }
+}
+
+.contact-info-wrapper {
+  display: flex;
+  flex-direction: column;
+  background: vars.$surface-container-low;
+  padding: 2rem;
+  min-width: 12.5rem;
+  max-width: 18.75rem;
+  border-right: 1px solid vars.$outline-variant;
```

Comment:
Put the border in the people profile on the left of the main content. That way you don't have to stretch the contact info to fill the entire height